src/lib/executors/Node.ts

```diff
@@ -876,7 +899,7 @@ export default class Node extends Executor<NodeEvents, Config, NodePlugins> {
       uncoveredFiles.forEach(filename => {
         try {
           const code = readFileSync(filename, { encoding: 'utf8' });
-          this.instrumentCode(code, filename);
+          this.instrumentCode(code, filename, true);
```

Comment:
Should this always be true, or should it depend on the extension?

Reply:
It should (and now does) depend on the extension